onyo/lib/onyo.py

```diff
@@ -226,59 +345,95 @@ def _init(self, path: Path) -> None:
         if not path.parent.exists():
             raise FileNotFoundError(f"'{path.parent}' does not exist.")
 
-        # Note: Why is this a requirement? Why not only add what's missing in case of apparent re-init?
-        # cannot already be an .onyo repo
+        # Note: Why is this a requirement? Why not only add what's missing in
+        # case of apparent re-init? cannot already be an .onyo repo
         dot_onyo = Path(path, '.onyo')
         if dot_onyo.exists():
             raise FileExistsError(f"'{dot_onyo}' already exists.")
 
         self.git.maybe_init(path)
 
         # Note: pheewww - No. Installed resource needs to be found differently.
-        #       Who the hell is supposed to maintain that? One cannot simply move
-        #       this function without changing its implementation.
+        #       Who the hell is supposed to maintain that? One cannot simply
+        #       move this function without changing its implementation.
         skel_dir = Path(Path(__file__).resolve().parent.parent, 'skel')
 
         # populate .onyo dir
         shutil.copytree(skel_dir, self.dot_onyo)
 
         # add and commit
-        self.git.stage_and_commit(self.dot_onyo, message='Initialize as an Onyo repository')
+        self.git.stage_and_commit(self.dot_onyo,
+                                  message='Initialize as an Onyo repository')
         log.info(f'Initialized Onyo repository in {self.dot_onyo}/')
 
-    def is_onyo_path(self, path: Path) -> bool:
-        return path == self.dot_onyo or self.dot_onyo in path.parents or path.name.startswith('.onyo')  # see .onyoignore
+    def is_onyo_path(self,
+                     path: Path) -> bool:
+        """
+        Identify for an absolute `path` if it is inside the `.onyo/` directory.
+
```

Comment:
```suggestion
        """
        Determine whether an absolute `path` is used by onyo internally.

        Currently anything underneath `.onyo/` as well as anything named `.onyo*` is considered an onyo path.
```

See also: respective test  and parameter descriptions below.
Note, that not only does it match `.onyo/**` and `**/.onyo*`, but more generally the description suggested by me above is broader and allows to enhance that arbitrarily. The details can go into the "long description" below the short statement of what that function is/does.